File: Célia_LingProgram/ListaJavaFacul /src/L1_Exer02_Idade/Idade.java
package L1_Exer02_Idade;

import java.util.Scanner;

public class Idade {
    public static void main(String[] args) {

        String nome;
        Integer anoNasc;
        Integer idadeFut;


        Scanner leitor = new Scanner(System.in);

        System.out.println("Exercícios 02 - Idade\n");
        System.out.println("Qual o seu nome?");
        nome = leitor.nextLine();
        System.out.println("Olá, " +nome+ "! Qual o ano de seu nascimento?");
        anoNasc = leitor.nextInt();
        idadeFut = 2030 - anoNasc;
        System.out.println("Em 2030 você terá " +idadeFut+ " anos");

    }
}

File: Célia_LingProgram/ListaJavaFacul/src/L1_Exer02_Idade/Idade.kt
package L1_Exer02_Idade

import java.util.*

//Crie uma classe chamada “Idade” Java que:
//A. Seja executável.
//B. Solicite que o usuário digite o seu nome.
//C. Em seguida, o programa deverá exibir a mensagem:
//“Olá, Fulano! Qual o ano de seu nascimento?”.
//D. Depois, o programa deverá exibir:
//“Em 2030 você terá 25 anos” (sendo Fulano o nome digitado pelo
//usuário e xx a idade
//do usuário).

fun main() {
    val reader = Scanner(System.`in`)

    println("IDADE")
    println("Digite seu nome: ")
    val nome: String = reader.nextLine()
    println("Olá, " + nome + "! Qual o ano do seu nascimento?")
    val anoNasc: Int = reader.nextInt()
    println("Em 2030 voce terá " + (2030 - anoNasc) + " anos")

}